src/main.cpp

```diff
@@ -40,7 +40,11 @@
 
 #include "binding.h"
 
+#ifdef(__WINDOWS__)
+#include "resource.h"
+#else
 #include "icon.png.xxd"
+#endif
```

Comment:
Then change the include to only happen on Linux, so it's consistent (it's just 1kb, but I don't care either way).